aspnetcore/index.yml

```diff
@@ -0,0 +1,455 @@
+          itemType: get-started
+          text: "ASP.NET Core MVC with Entity Framework Core"
+        - url: https://docs.microsoft.com/en-us/learn/modules/build-web-api-net-core/
+          itemType: get-started
+          text: "Create a page-focused web UI that consumes a RESTful web API"
+
+# conceptualContent section (optional)
+conceptualContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  title: Develop ASP.NET Core apps # < 60 chars (optional)
+  summary: "Choose interactive web apps, web API, MVC-patterned apps, real-time apps, and more"
+  items:
+    # Card
+    - title: "Interactive client-side Blazor apps"
+      summary: Develop with reusable UI components that can take advantage of WebAssembly for near-native performance
+      links:
+        - url: blazor/index.md
+          itemType: overview
+          text: "Overview"
+        - url: blazor/get-started.md
+          itemType: get-started
+          text: "Create your first Blazor app"
+        - url: tutorials/build-your-first-blazor-app.md
+          itemType: get-started
+          text: "Build your first Blazor app with reusable components"
+        - url: blazor/hosting-models.md
+          itemType: concept
+          text: "Blazor hosting models"
+    # Card
+    - title: RESTful web API apps
+      summary: Develop RESTful HTTP services with ASP.NET Core web API
+      links:
+        - url: web-api/index.md
+          itemType: overview
+          text: "Overview"
+        - url: tutorials/first-web-api.md
+          itemType: get-started
+          text: "Create your first web API with ASP.NET Core"
+        - url: tutorials/web-api-help-pages-using-swagger.md
+          itemType: tutorial
+          text: "Generate web API help pages with Swagger / OpenAPI"
+        - url: web-api/action-return-types.md
+          itemType: concept
+          text: "Controller action return types"
+        - url: web-api/advanced/formatting.md
+          itemType: concept
+          text: "Format response data"
+        - url: web-api/handle-errors.md
+          itemType: concept
+          text: "Handle errors"
+        - url: tutorials/web-api-javascript.md
+          itemType: tutorial
+          text: "Call an ASP.NET Core web API with JavaScript"
+      # footerLink (optional)
+      footerLink:
+        url: web-api/index.md
+        text: "See more"
+    # Card
+    - title: Page-focused web UI with MVC
+      summary: "Develop web apps using the Model-View-Controller design pattern"
+      links:
+        - url: mvc/overview.md
+          itemType: overview
+          text: "Overview"
+        - url: tutorials/first-mvc-app/index.md
+          itemType: get-started
+          text: "Create your first ASP.NET Core MVC app"
+        - url: tutorials/first-mvc-app/adding-controller.md
+          itemType: tutorial
+          text: "Add a controller"
```

Comment:
Don't add the entire series here. This should be a list of fairly independent topics.

Reply:
I removed these in a patch.  Thanks Rick.